docs/bidders/beachfront.md

```diff
@@ -0,0 +1,2 @@
+# Beachfront bidder
```

Comment:
Do you have docs to add here? If not, just delete it.

Reply:
I've added docs and parameter testing.